main.nf

```diff
@@ -613,27 +615,130 @@ process dia_spectral_library_search {
         -mz_extraction_window_unit ${params.mz_extraction_window_unit} \\
         -mz_extraction_window_ms1_unit ${params.mz_extraction_window_ms1_unit} \\
         -rt_extraction_window ${params.rt_extraction_window} \\
-        -min_upper_edge_dist ${params.min_upper_edge_dist} \\
+	-extra_rt_extraction_window ${params.extra_rt_extraction_window} \\
+	-ion_mobility_window ${params.ion_mobility_window} \\
+	-im_extraction_window_ms1 ${params.im_extraction_window_ms1} \\
+	-use_ms1_ion_mobility ${params.use_ms1_ion_mobility} \\
+	-matching_window_only ${params.matching_window_only} \\
+	-irt_mz_extraction_window ${params.irt_mz_extratcion_window} \\
+	-irt_mz_extraction_window_unit ${params.irt_mz_extraction_window_unit} \\
+        -irt_im_extraction_window ${params.irt_im_extraction_window} \\
+	-split_file_input ${params.split_file_input} \\
+	-use_elution_model_score ${params.use_elution_model_score} \\
+	-mz_correction_function ${params.mz_correction_function} \\
+	-extraction_function ${params.extraction_function} \\
+	-outer_loop_threads ${params.outer_loop_threads} \\
+	-ms1_isotopes ${params.ms1_isotopes} \\
+	-log ${params.log} \\
+	-debug ${params.debug} \\
```

Comment:
Please do not add pipeline parameters called `--log` and `--debug`, as they are too generic.